test/integration/features/user_creates_from_url.spec.js

```diff
@@ -0,0 +1,204 @@
+const projectHelpers = require('../helpers/project');
+const inputsHelpers = require('../helpers/inputs');
+const CreateFromURLPage = require('../page-objects/createFromURL').CreateFromURLPage;
+const CreateProjectPage = require('../page-objects/createProject').CreateProjectPage;
+const CatalogPage = require('../page-objects/catalog').CatalogPage;
+const nodeMongoTemplate = require('../fixtures/nodejs-mongodb');
+const centosImageStream = require('../fixtures/image-streams-centos7.json');
+
+describe('authenticated e2e-user', function() {
+
+  let project = projectHelpers.projectDetails();
+  let namespaceForFixtures = "template-dumpster";
+
+  let setupEnv = function() {
+    // add namespace to create whitelist for template and image stream
+    addExtension(`
+      (function() {
+        // Update whilelist:
+        window.OPENSHIFT_CONSTANTS.CREATE_FROM_URL_WHITELIST = ['openshift', '${namespaceForFixtures}'];
+      })();
+    `);
+    let fixturesProject = {name: namespaceForFixtures};
+    let createProjectPage = new CreateProjectPage(fixturesProject);
+    createProjectPage.visit();
+    createProjectPage.createProject();
+    let catalogPage = new CatalogPage(fixturesProject);
+    // - add an image stream to that namespace
+    catalogPage.visit();
+    catalogPage.processImageStream(JSON.stringify(centosImageStream));
+    // - add a template to that namespace
+    catalogPage.visit();
+    catalogPage.saveTemplate(JSON.stringify(nodeMongoTemplate));
+  };
+
+  beforeAll(function() {
+    h.commonSetup();
+    h.login();
+    projectHelpers.deleteAllProjects();
+    setupEnv();
+  });
+
+  afterAll(function() {
+    projectHelpers.deleteAllProjects();
+    h.afterAllTeardown();
+  });
+
+  describe('create from URL', function() {
+
+    describe('using an image stream and image stream tag supplied as query string params', function() {
+
+      let name = 'nodejs-edited';
+      let sourceURI = 'https://github.com/openshift/nodejs-ex.git-edited';
+      let sourceRef = 'master-edited';
+      let contextDir = '/-edited';
+      let qs = '?imageStream=nodejs&imageTag=4&name=' + name + '&sourceURI=' + sourceURI + '&sourceRef=' + sourceRef + '&contextDir=' + contextDir + '&namespace=' + namespaceForFixtures;
+      let uri = 'project/' + project.name + 'create/fromimage' + qs;
+      let heading = 'Node.js 4';
+      let words = project.name.split(' ');
+      let timestamp = words[words.length - 1];
+
+      it('should display details about the the image', function() {
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        matchersHelpers.expectHeading(heading);
+      });
+
+      it('should load the image stream in to a newly created project', function(){
+        let createFromURLPage = new CreateFromURLPage();
+        createFromURLPage.visit(qs);
+        // NOTE: if the tabs are on the page because there are existing projects, click the 'Create New Project' tab first
```

Comment:
I think we can drop this comment since we don't have the `if` statement here anymore?